src/md-collection-loader/description.js

```diff
@@ -0,0 +1,39 @@
+import remark from "remark"
+import stripMd from "strip-markdown"
+import { prune }  from "underscore.string"
+
+const defaultOpts = {
+  pruneLength: 140,
+  pruneString: "…",
```

Comment:
Is the pruneString considered when counting and splitting value?

"Limited to X chars" => "Limited to X char…"  (this is a stupid cut :D)